.stylelintrc.json

```diff
@@ -6,7 +6,7 @@
     "custom-property-pattern": null,
     "function-no-unknown": [
       true,
-      { "ignoreFunctions": ["color-mod", "alpha", "blackness"] }
+      { "ignoreFunctions": ["alpha", "blackness"] }
```

Comment:
all of these functions should be removed as `alpha` and `blackness` are used as a part of `color-mix`

```suggestion
```

can you ensure the linting still works even after this change?

Reply:
Good catch, didn't notice that! Will remove and check the stylelint.

Reply:
Confirmed that Stylelint still works.